extensions/rich_text_components/rte-output-display.component.ts

```diff
@@ -31,6 +31,7 @@ type PortalTree = (TemplatePortal<unknown> | PortalTree) [];
 export class RteOutputDisplayComponent implements AfterViewInit {
   // Native HTML elements.
   @ViewChild('p') pTagPortal: TemplateRef<unknown>;
+  @ViewChild('h1') h1TagPortal: TemplateRef<unknown>;
```

Comment:
Why `unknown`?

Reply:
Since it is a structural directive that can be used anywhere, we use `unknown`